email-ballerina/commons.bal

```diff
@@ -79,24 +77,17 @@ public type Attachment record {|
 
 # Secure Socket configuration.
 #
-# + certificate - Server certificate
+# + cert - Server certificate path
 # + protocol - SSL or TLS protocol
 # + ciphers - Ciper used
 # + verifyHostname - Enable hostname verification
-public type SecureSocket record {|
-    Certificate certificate;
+public type ClientSecureSocket record {|
+    string cert;
     Protocol? protocol = ();
     string[]? ciphers = ();
     boolean verifyHostname = true;
 |};
 
-# Certificate configuration.
-#
-# + path - Certificate file location
-public type Certificate record {|
-    string path;
-|};
-
 # Transport security protocol.
```

Comment:
And shall we remove this record with the above comment?

Reply:
Removed.